src/nodes/nora-sousvide.ts

```diff
@@ -0,0 +1,56 @@
+import { TemperatureControlDevice } from '@andrei-tatar/nora-firebase-common';
+import { ConfigNode, NodeInterface } from '..';
+import { registerNoraDevice } from './util';
+
+module.exports = function (RED: any) {
+    RED.nodes.registerType('noraf-sousvide', function (this: NodeInterface, config: any) {
+        RED.nodes.createNode(this, config);
+
+        const noraConfig: ConfigNode = RED.nodes.getNode(config.nora);
+        if (!noraConfig?.valid) {
+            return;
+        }
+
+        const deviceConfig: Omit<TemperatureControlDevice, 'id'> = {
+            type: 'action.devices.types.SOUS_VIDE',
+            name: { name: config.devicename },
+            traits: ['action.devices.traits.TemperatureControl'],
+            state: {
+                online: true,
+                temperatureSetpointCelsius: 50,
+                temperatureAmbientCelsius: 25,
+            },
+            attributes: {
+                queryOnlyTemperatureControl: config.queryOnlyTemperatureControl,
```

Comment:
same, name in the config is different:
`queryOnlyTemperatureControl: config.queryOnly,`